src/script_lua.c

```diff
@@ -596,7 +596,7 @@ int luaError(lua_State *lua) {
 
 /* Reply to client 'c' converting the top element in the Lua stack to a
  * server reply. As a side effect the element is consumed from the stack.  */
-static void luaReplyToRedisReply(client *c, client* script_client, lua_State *lua) {
+static void luaReplyToServerReply(client *c, client* script_client, lua_State *lua) {
```

Comment:
ServerReply = RESP?

Maybe we lost the meaning when renaming to ServerReply? Another option could be luaReplyToRespReply.

Nah... I don't think it's a source of confusion. Ignore this comment. :)